README.md

```diff
@@ -1,41 +1,201 @@
 # kotlin-support
+
 User Language Support for the Kotlin Programming Language
 
+## EventSourcing example of use
+
+### Define your proto
+
+```proto
+syntax = "proto3";
```

Comment:
@sleipnir In order to ensure that this does not decay over time, perhaps use Paradox includes to pull this in? https://developer.lightbend.com/docs/paradox/current/directives/snippets.html

Reply:
Ok